calliar_server/server/static/utils.js

```diff
@@ -38,4 +38,14 @@ function preprocess(name)
     }
 
     return [text, outText]
+}
+
+function getPoints(path){
+    var points = []
+    var segments = path.segments
+
+    for(i = 0; i< segments.length; i++){
```

Comment:
would it be better to use the `ForEach` syntax here?
```javascript
segments.forEach(segment => points.push([semgnet.point.x,segment.point.y])
```

Reply:
This is an enhancement. It can be even done later.

Reply:
fixed